src/features/bravery/components/ChampionImage.jsx

```diff
@@ -20,20 +21,56 @@ export function ChampionImage({ imageUrl, championObject }) {
       : delete updatedChampionList[championObject.id];
     actions.setSelectedChampions(updatedChampionList);
   };
+  const handleMouseEnter = () => {
+    setIsHovered(true);
+  };
+  const handleMouseLeave = () => {
+    setIsHovered(false);
+  };
 
   return (
     <>
-      <Avatar
-        flex="0 1 calc(100% / 8 - .5rem)"
-        height="auto"
-        minWidth="55px"
-        onClick={toggleSelectedChampion}
-        filter={
-          !state.selectedChampions?.[championObject.id] && "grayscale(100%)"
-        }
-        src={`${CHAMPION_IMAGE_BASE_URL}${imageUrl}`}
-        borderRadius={2}
-      />
+      <Tooltip label={championObject.name} placement="auto">
+        <Box
+          position="relative"
+          onMouseEnter={handleMouseEnter}
+          onMouseLeave={handleMouseLeave}
```

Comment:
```suggestion
          onMouseEnter={() => setIsHovered(true)}
          onMouseLeave={() => setIsHovered(false)}
```
can get rid of some lines by calling the functions directly.